pkg/metrics/metrics.go

```diff
@@ -16,85 +16,60 @@ package metrics
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const (
-	// MetricsEndpoint is the port to export metrics on
-	MetricsEndpoint = ":8080"
-)
-
 var (
-	metricTotalAWSAccounts = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAWSAccounts = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_aws_accounts",
 		Help: "Report how many accounts have been created in AWS org",
 	}, []string{"name"})
-	metricTotalAccountCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_account_crs",
 		Help: "Report how many account CRs have been created",
 	}, []string{"name"})
-	metricTotalAccountCRsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountCRsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_accounts_crs_unclaimed",
 		Help: "Report how many account CRs are unclaimed",
 	}, []string{"name"})
-	metricTotalAccountCRsClaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountCRsClaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_accounts_crs_claimed",
 		Help: "Report how many account CRs are claimed",
 	}, []string{"name"})
-	metricTotalAccountCRsFailed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountCRsFailed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_accounts_crs_failed",
 		Help: "Report how many account  CRs are failed",
 	}, []string{"name"})
-	metricTotalAccountClaimCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountClaimCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_aws_account_claim_crs",
 		Help: "Report how many account claim CRs have been created",
 	}, []string{"name"})
-	metricPoolSizeVsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricPoolSizeVsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_pool_size_vs_unclaimed",
 		Help: "Report the difference between the pool size and the number of unclaimed account CRs",
 	}, []string{"name"})
-	metricTotalAccountPendingVerification = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	MetricTotalAccountPendingVerification = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "aws_account_operator_total_aws_account_pending_verification",
 		Help: "Report the number of accounts waiting for enterprise support and EC2 limit increases in AWS",
 	}, []string{"name"})
 
-	metricsList = []prometheus.Collector{
-		metricTotalAWSAccounts,
-		metricTotalAccountCRs,
-		metricTotalAccountCRsUnclaimed,
-		metricTotalAccountCRsClaimed,
-		metricTotalAccountCRsFailed,
-		metricTotalAccountClaimCRs,
-		metricPoolSizeVsUnclaimed,
-		metricTotalAccountPendingVerification,
+	MetricsList = []prometheus.Collector{
```

Comment:
This variable seems to be unused in the operator, is it something you need from the `operator-custom-metrics` pkg?

Reply:
This will be used in a future change of the custom metrics package coming in the next week or so